rest-client/src/test/java/org/sagebionetworks/bridge/rest/ClientManagerTest.java

```diff
@@ -96,6 +96,10 @@ public void canOverrideHostUrl() {
         Config config = new Config();
         config.set(Props.HOST, "https://aws.bridgeserver.com");
         config.set(Environment.PRODUCTION);
+
+        config.set(Props.STUDY_IDENTIFIER, "studyId");
```

Comment:
Is this change OK? This allows the test to run without setting env vars or bridge-sdk.properties, I think it's still verifying that the host URL can be overridden and not defeating the point of the test.